src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_PatternSwitchStatement.cs

```diff
@@ -320,7 +320,7 @@ private void AddBindings(ArrayBuilder<BoundStatement> sectionBuilder, ImmutableA
                 {
                     var source = kv.Key;
                     var dest = kv.Value;
```

Comment:
We already had this discussion in context of [https://github.com/dotnet/roslyn/pull/13535](https://github.com/dotnet/roslyn/pull/13535). There is a simple rule that implementation of LocalRewriter should follow - every node that is a part of the input bound tree should be lowered. Following this simple rule doesn't create any correctness problems, but at the same time prevents us from introducing hard to find bugs and makes local rewriter robust to future changes in the pipeline.   

---
In reply to: [80815417](https://github.com/dotnet/roslyn/pull/13999#discussion_r80815417) [](ancestors = 80815417,80800519)